File: src/gui/ForwardingPane.java
package gui;

import javafx.scene.control.Label;
import javafx.scene.layout.*;
import javafx.scene.paint.Paint;

/**
 * Created by Eric Tsang on 11/8/2015.
 */
public class ForwardingPane extends VBox
{
    public ForwardingPane()
    {
        super();

        // add a red, dashed 3 pixel, rounded thick border about the layout
        setBorder(new Border(new BorderStroke(Paint.valueOf("0f0"),BorderStrokeStyle.DASHED,new CornerRadii(10),new BorderWidths(3))));

        // label
        Label label = new Label();
        label.setText("ForwardingPane");
        getChildren().add(label);
    }
}

File: src/gui/ForwardingPane.kt
package gui

import javafx.scene.control.Label
import javafx.scene.layout.Border
import javafx.scene.layout.BorderStroke
import javafx.scene.layout.BorderStrokeStyle
import javafx.scene.layout.BorderWidths
import javafx.scene.layout.CornerRadii
import javafx.scene.layout.VBox
import javafx.scene.paint.Paint

/**
 * Created by Eric Tsang on 11/8/2015.
 */
class ForwardingPane:VBox()
{
    init
    {
        // add a red, dashed 3 pixel, rounded thick border about the layout
        border = Border(BorderStroke(Paint.valueOf("0f0"),BorderStrokeStyle.DASHED,CornerRadii(10.0),BorderWidths(3.0)))

        // label
        val label = Label()
        label.text = "ForwardingPane"
        children.add(label)
    }
}
